placeholders/src/main/java/org/eclipse/ditto/placeholders/ImmutableTimePlaceholder.java

```diff
@@ -42,8 +44,12 @@ final class ImmutableTimePlaceholder implements TimePlaceholder {
     private static final String NOW_PLACEHOLDER = "now";
     private static final String NOW_EPOCH_MILLIS_PLACEHOLDER = "now_epoch_millis";
 
-    private static final List<String> SUPPORTED = Collections.unmodifiableList(
-            Arrays.asList(NOW_PLACEHOLDER, NOW_EPOCH_MILLIS_PLACEHOLDER));
+    private static final String TRUNCATE_START;
+    private static final String TRUNCATE_END = "]";
+
+    static {
```

Comment:
Is there a need for this static block?

Reply:
Yeah, that was kinda strange. I could not declare `"["` directly to a static constant.
IbtelliJ was complaining.
But when I moved the assignment into the static block, it worked..

Could also be an IDE bug.. 😕